drizzlepac/align.py

```diff
@@ -398,14 +434,16 @@ def perform_align(input_list, catalog_list, num_sources, archive=False, clobber=
             table = alignment_table.extracted_sources[imgname]
 
             # Get the location of the current image in the filtered table
-            index = np.where(alignment_table.filtered_table['imageName'] == imgname)[0][0]
+            index = np.where(alignment_table.filtered_table["imageName"] == imgname)[0][
+                0
+            ]
 
```

Comment:
Lines 437-439 represent one of the odd formatting instances which is just too silly.  Please fix this manually.

Reply:
Ahh, agreed. Good catch.